radis/lbl/broadening.py

```diff
@@ -2248,6 +2248,23 @@ def _broaden_lines(self, df):
                 N = int(len(df) * len(wavenumber) / chunksize) + 1
                 # Too big may be faster but overload memory.
                 # See Performance for more information
+
+                # Raise performance warning if the chunks are bigger
+                # than the number of lines.
+                try:
+                    assert N < len(df)
+                except AssertionError:
```

Comment:
what do you think of an `if` syntax instead ? Seems more compact to me

Reply:
Yes, I agree that it would be more compact. I'll change the syntax to if-raise